components/01-atoms/buttons/_buttons.scss

```diff
@@ -0,0 +1,86 @@
+$button-radii: (
+  'soft': var(--button-radius-soft),
+  'pill': 100vw,
```

Comment:
Should this be converted to a token value instead? `$layout-width-full` ?

Reply:
I considered this, but the "pill" (`100vw`) radius is only used by buttons, and should not be accessible to anything else. If I add it to the `radius` array, then it shows up as a border-radius option for things like cards too, and I don't want that. If it does come up again, maybe I'll see about moving it out of this file, but for now, I think this is the right place for the definition.

(Also, `layout-width-full` is 100% which is very different from 100vw)

oh... I actually just realized I should be using 100vmax... I'll make that change.

Reply:
Gotcha. All makes sense here and I agree. I just saw a static value being set and wanted to confirm this was the ideal setup.